doc/user/content/concepts/views.md

```diff
@@ -15,104 +15,119 @@ aliases:
+query. Views can be referenced across [clusters](/concepts/clusters/).
 
-See also:
+During view creation, the underlying query is not executed , and the view
+results are not persisted in durable storage.
 
-- [`CREATE VIEW`](/sql/create-view)
+```mzsql
+CREATE VIEW my_view_name AS
+  SELECT ... FROM ...  ;
+```
 
-### Indexes and non-materialized views
+<red>However</red>, in Materialize, you can [index](/concepts/indexes/) a view
+to **maintain and incrementally update** view results in memory within a
+cluster.
 
-Indexes can improve query performance. In Materialize, non-materialized views
-can be [indexed](/concepts/indexes/). Indexing a non-materialized view causes
-view results to be **maintained and incrementally updated in memory** within
-that [cluster](/concepts/clusters/). The in-memory up-to-date results are
-accessible to queries within the cluster, even for queries that do not use the
-index key(s).
+```mzsql
+CREATE INDEX idx_on_my_view ON my_view_name(...) ;
+```
 
-Indexes are local to a cluster.
+Queries within that cluster can use the index to access view results from
+memory.  See [Indexes and views](#indexes-and-views) for more information.
 
 See also:
 
-- [Indexes](/concepts/indexes/)
-- [`CREATE INDEX`](/sql/create-index/)
+- [`CREATE VIEW`](/sql/create-view)  for complete syntax information
+- [`CREATE INDEX`](/sql/create-index/)  for complete syntax information
 
-### Clusters and non-materialized views
+### Indexes and views
 
-A non-materialized view can be referenced in any [cluster](/concepts/clusters/).
-However, if the non-materialized view is indexed, in-memory results are only
-accessible to queries within the cluster.
+In Materialize, views can be [indexed](/concepts/indexes/).
+
+```mzsql
+CREATE INDEX idx_on_my_view ON my_view_name(...) ;
+```
+
+Indexes on views both **maintain and incrementally update view results in
+memory** within that [cluster](/concepts/clusters/). The in-memory up-to-date
+results are accessible to queries within the cluster, even for queries that do
+not use the index key(s).
+
+See also:
 
-Indexes are local to a cluster.  Queries in a different cluster cannot use the
-indexes in another cluster.
+- [Indexes](/concepts/indexes)
+- [Optimization](/transform-data/optimization)
+- [`CREATE INDEX`](/sql/create-index/)  for complete syntax information
 
 ## Materialized views
 
-A materialized view, like a non-materialized view, saves a query under a name to
-provide a shorthand for referencing the query. But, unlike a non-materialized
-view, the query is executed during the view creation, and a materialized view
-**persists and incrementally updates** its results in durable storage.
+In Materialize, a materialized view is a view whose underlying query is executed
+during the view creation, and the view results are both **persisted and
+incrementally updated** in durable storage. Materialized views can be referenced across [clusters](/concepts/clusters/).
```

Comment:
Similar to what we discussed for views, materialized views are a known concept in databases. We could emphasize what makes them different in Materialize after introducing the basic concept (like we're doing for views with indexes).